content/04.methods.md

```diff
@@ -169,6 +169,32 @@ All merged `SingleCellExperiment` objects were converted to `AnnData` objects an
 If a merged `SingleCellExperiment` object contained any ADT data, the RNA and ADT data were exported and saved separately as RNA (`_rna.h5ad`) and ADT (`_adt.h5ad`) objects.
 In contrast, if a merged `SingleCellExperiment` object contained HTO data due to the presence of any multiplexed libraries in the merged object, the HTO data was removed from the `SingleCellExperiment` object and not included in the exported `AnnData` object.
 
+### Analysis of bulk RNA-seq data
+
+#### Data preparation
+
+We identified solid tumor samples with both bulk and single-cell (or single-nuclei) RNA-seq data in the ScPCA Portal for analysis, with multiplexed samples excluded (N=105).
+We further identified low-quality samples based on visual inspection of quality control reports and removed (N=8), leaving a total of 97 samples across five ScPCA projects for analysis.
+
+For each project, we transformed and normalized with respect to library size bulk counts matrices for all samples using `DESeq2::rlog()` [@doi:10.1186/s13059-014-0550-8].
+We obtained pseudobulk counts by summing raw single-cell counts for each sample, and similarly transformed each project's resulting counts matrix with `DESeq2::rlog()`.
+We filtered out genes which were not observed in either the bulk or pseudobulk raw counts matrices before subsequent analysis.
+For each project, we then used the `lme4` [@doi:10.18637/jss.v067.i01] R package to construct a linear model predicting bulk from pseudobulk counts considering a random effect for sample id: `bulk ~ pseudobulk + (1|sample_id)`.
+
+#### Overrepresentation analysis
+
+To identify cell types that might be overrrepresented in one modality compared to the other, we first identified gene sets associated with each cell type of interest, defined as each project's consensus cell types.
```

Comment:
I think we want to separate the selection of the cell types we are looking at for a project from the gene sets that are tested.  So first we select cell types of interest using conensus cell types for the project. Then we identify the gene set for each cell type. I think that will clarify?

(my suggestion below is only about removing the extra "r")
```suggestion
To identify cell types that might be overrepresented in one modality compared to the other, we first identified gene sets associated with each cell type of interest, defined as each project's consensus cell types.
```

Reply:
> (my suggestion below is only about removing the extra "r")

I have two very mediocre, not fully formed jokes:
- so many r's this could be an r package!
- yes of course, the r's are overrepresented